astropy/table/soco.py

```diff
@@ -73,6 +73,9 @@ class SCEngine:
     '''
 
     def __init__(self, data, row_index, unique=False):
+        if not HAS_SORTEDCONTAINERS:
+            raise ImportError("`sortedcontainers` is needed for using `SCEngine`")
```

Comment:
No need for Sphinx formatting in the error message here.
```suggestion
            raise ImportError("sortedcontainers is needed for using SCEngine")
```